packages/react-conformance/src/utils/merge.tsx

```diff
@@ -0,0 +1,41 @@
+import * as _ from 'lodash';
+
+// tslint:disable-next-line:no-any
```

Comment:
You'll need to merge with master and switch these to `eslint-disable @typescript-eslint/no-explicit-any`

Reply:
sorry forgot about that